ChangeLog

```diff
@@ -35,6 +35,10 @@ New deprecations
    * Deprecate mbedtls_ctr_drbg_update and mbedtls_hmac_drbg_update
      in favor of functions that can return an error code.
 
+Bugfix
+   * Fix double initialisation of ECC hardware that made some accelerators
```

Comment:
s/initialisation/initialization, here and in commit messages too

https://docs.internal.mbed.com/docs/master/style/vocabulary-and-spelling.html

Reply:
Amended the corresponding commits.